### ISSUE TITLE
Add standards wheel calibration to XAS scan initialization actions

### ISSUE THREAD

**Issue by helfrij** [acquaman/acquaman#1672]
The standards wheel motor on Side is quite sensitive and may become un-calibrated if someone touches or bumps it. There is no reliable feedback system for determining whether or not the motor is currently calibrated. As this device is in an area where users could accidentally interfere with the motor, and all XAS scans automatically attempt to move the standards wheel to the element being scanned, it may be a good idea to include a calibration routine for the motor in scan initialization.

There is a pv that triggers the calibration routine: SMTR1607-6-I22-19:calibRun. A control for this pv could be added to CLSStandardsWheel, and triggering this control added to the scan initialization actions.
- Check with David B. that the calibration routine does what I think it does--just needs to be run, no further action required.
- I'm not sure this is a feature that would be available generally. It may make more sense to add this to a BioXASStandardsWheel. Thoughts @dretrex?

**Comment by dretrex** [comment]
Yeah, I agree that it should be added to the BioXASStandardsWheel class.  As to the comment of adding it to the initialization, I don't think it should be a part of the general XAS scan initialization.  What we could do is have something that adds it before a set of scans, but I think doing the calibration after every scan is overkill.

**Comment by helfrij** [comment]
Talked to David B. today, the calibration routine isn't fully implemented yet. I'm not sure it's a good idea to worry too much about adding calibration functionality to scans until it's ready.

**Comment by dretrex** [comment]
Okay, sounds good.

**Comment by helfrij** [comment]
I think the calibration routine has now been implemented and tested many times. There was worry that calibrating the standards wheel motor with every scan is somewhat overzealous and unnecessarily time-consuming. @dretrex, do you happen to remember if we came up with a compromise?
